docker-compose.yml

```diff
@@ -4,6 +4,13 @@ redis:
   image: redis:2.8
 rabbitmq:
   image: rabbitmq:3.5
+solr:
+  image: makuk66/docker-solr:4.10.4
+  volumes:
+    # hydroshare repository
+    - "/home/hydro/hydroshare:/hydroshare"
```

Comment:
The definitive changes should be made to the docker-compose.template file as it is used to generate the docker-compose.yml on each hsctl call that involves the pre_flight function based on the contents of the hydroshare-config.yaml file settings.